test/common/initModule.js

```diff
@@ -182,22 +178,22 @@ var modulesLoader = new function () {
 	this.initAllModules = function (cb, scope) {
 		this.clear();
 		this.initModules([
-			{accounts: require('../../modules/accounts')},
-			{blocks: require('../../modules/blocks')},
-			{crypto: require('../../modules/crypto')},
-			{delegates: require('../../modules/delegates')},
-			{loader: require('../../modules/loader')},
-			{multisignatures: require('../../modules/multisignatures')},
-			{peers: require('../../modules/peers')},
-			{signatures: require('../../modules/signatures')},
-			{system: require('../../modules/system')},
-			{transactions: require('../../modules/transactions')},
-			{transport: require('../../modules/transport')}
+			{accounts: 					alias.require('modules/accounts')},
```

Comment:
prefer the regular indents

Reply:
I have reverted the files to use standard indentation and pushed it to the repo.